tools/ci_build/github/azure-pipelines/templates/rocm.yml

```diff
@@ -5,18 +5,41 @@ parameters:
 - name: RocmVersion
   type: string
 
-
+- name: BuildConfig
+  type: string
+  default: 'Release'
 
 jobs:
-- job: wheels_python_${{ replace(parameters.PythonVersion,'.','_') }}_rocm_${{ replace(parameters.RocmVersion,'.','_') }}
+- job: wheels_python_${{ replace(parameters.PythonVersion,'.','_') }}_rocm_${{ replace(parameters.RocmVersion,'.','_') }}_${{ parameters.BuildConfig }}
   workspace:
     clean: all
   timeoutInMinutes: 180
   pool: Ubuntu-2004-rocm-aiinfra
   variables:
-    PythonVersion: ${{ parameters.PythonVersion }}
+    - name: PythonVersion
```

Comment:
Actually, you don't need to redefine the following two parameters as variables:
```yml
    - name: PythonVersion
      value: ${{ parameters.PythonVersion }}
    - name: BuildConfig
      value: ${{ parameters.BuildConfig }}
```
You can directly use them in whatever place you need. 

You may think variables vs parameters is like variables vs macros in C/C++. For example, if you already have a macro like:
```c++
#define PI 3.14
```
You don't need to write
```c++
float pi = PI;
float r  = 3;
float area = pi * pi * r;
```
You can directly use the macro in the last statement. 

As the same, template parameters in yaml files are compile-time constant.  I would prefer using them instead of variables.

Reply:
Remove one redundant variable, and keep `PytorchVersion` as variable, which will be passed to template set-python-manylinux-variables-step.yml.